src/SFA.DAS.AssessorService.Application.Api.Client/Clients/OrganisationsApiClient.cs

```diff
@@ -52,6 +62,14 @@ public async Task<OrganisationResponse> Get(string ukprn)
             }
         }
 
+        public async Task<Organisation> Get(Guid organisationId)
+        {
+            using (var request = new HttpRequestMessage(HttpMethod.Get, $"/api/v1/organisations/organisation/{organisationId}"))
```

Comment:
This one looks a bit strange too. Could we not use: _/api/v1/organisations/{organisationId}_

Reply:
This is an existing endpoint.

Reply:
Fair enough. Although we need to start addressing these badly named end points at some point 💩